custom/css/style.css

```diff
@@ -1298,6 +1298,28 @@ ul.faq-answer li a:hover {
     padding-bottom: 10px;
 }
 
+#sponsors-content h3 {
+    margin-top: 20px;
+    font-size:40px;
+}
+
+@media screen and (min-width: 900px) {
+    .gojek {
+        margin-top: 10px;
+    }
+
+    .pramati {
+        margin-top: 35px;
+    }
+
+    .livehealth {
+        margin-top: 35px;
+    }
+    .rechonsys {
```

Comment:
Please fix the name `reckonsys` just to maintain consistency

Reply:
ACK